packages/@okta/vuepress-site/docs/release-notes/2020-02-1/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Okta API Products Release Notes
+---
+
+## 2020.02.1
+
+| Change                                             | Expected in Preview Orgs |
+|----------------------------------------------------|--------------------------|
+| [Bugs Fixed in 2020.02.1](#bug-fixed-in-2020-02-1) | February 19, 2020        |
```

Comment:
Link needs an "s" in bugs:
[Bugs Fixed in 2020.02.1](#bugs-fixed-in-2020-02-1) | February 19, 2020        |

Reply:
Thanks, fixed.